Dockerfile/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM osrf/ros:foxy-desktop
+
+# Make sure everything is up to date before building from source
+RUN apt-get update \
+  && apt-get upgrade -y \
+  && apt-get update && apt-get install -q -y --no-install-recommends \
+    dirmngr \
+    gnupg2 \
+    lsb-release \
+    wget \
+    curl \
+  && apt-get clean
+
+RUN sh -c 'echo "deb http://packages.osrfoundation.org/gazebo/ubuntu-stable `lsb_release -cs` main" > /etc/apt/sources.list.d/gazebo-stable.list' && \
+  wget https://packages.osrfoundation.org/gazebo.key -O - | apt-key add - && \
+  sh -c 'echo "deb http://packages.ros.org/ros2/ubuntu `lsb_release -cs` main" > /etc/apt/sources.list.d/ros2-latest.list' && \
+  curl -s https://raw.githubusercontent.com/ros/rosdistro/master/ros.asc | apt-key add - && \
+  apt-get update && apt-get install -q -y --no-install-recommends \
+    python3-colcon-ros \
+    python3-colcon-common-extensions \
+    python3-rosdep \
+    libignition-gazebo5-dev \
```

Comment:
Same here about removing packages.osrfoundation.org and `gazebo5` - this should work purely with packages.ros.org + rosdep

(if it doesn't, then it can't be bloomed in the ROS buildfarm)